daisy/image.go

```diff
@@ -116,6 +116,7 @@ type ImageInterface interface {
 	getRawDiskSource() string
 	setRawDiskSource(rawDiskSource string)
 	create(cc daisyCompute.Client) error
+	markCreated()
```

Comment:
Does this mean that the resource was created in general (perhaps it was found to exist already), or that it was explicitly created by this workflow? If the second, I'd suggest a more descriptive name for the field and method.

Also, where's the field defined?

Reply:
It's the 2nd. Let me name it "createdInWorkflow"